driver/driver.go

```diff
@@ -142,7 +142,7 @@ type ObjTool interface {
 
 	// Disasm disassembles the named object file, starting at
 	// the start address and stopping at (before) the end address.
-	Disasm(file string, start, end uint64) ([]Inst, error)
+	Disasm(file string, start, end uint64, intelSyntax bool) ([]Inst, error)
```

Comment:
I wonder if accepting string is better to accommodate future changes in supported syntax. ex: asm_syntax=intel

Reply:
I wouldn't do it - this makes code harder to refactor. We can revise the current approach when we have more options. E.g. this could take an option structure of something. But just an argument is fine for now.

Reply:
Ack.